### PR TITLE
Add method to avoid duplicated circuits

### PR DESCRIPTION
**Pull request by macartur**
Fix kytos/mef_eline#24

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -161,6 +165,21 @@ def create_circuit(self):
 
         return jsonify({"circuit_id": evc.id}), 201
 
+    def is_duplicated_evc(self, evc):
```
Comment (beraldoleal) [inline main.py]: implement this on `__eq__` please.

Comment (macartur) [reply main.py]: Fixed.

File: main.py [hunk 2]
```diff
@@ -148,6 +148,10 @@ def create_circuit(self):
         except ValueError as exception:
             return jsonify("Bad request: {}".format(exception)), 400
 
+        # verify duplicated evc
+        if self.is_duplicated_evc(evc):
+            return jsonify("Not Acceptable: This evc already exists."), 406
```
Comment (hdiogenes) [inline main.py]: It looks like the 406 error code has a [very specific meaning](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406):

> 406 Not Acceptable client error response code indicates that a response
> matching the list of acceptable values defined in Accept-Charset
> and Accept-Language cannot be served.

**409 Conflict** seems to be a [better alternative](https://stackoverflow.com/questions/3290182/rest-http-status-codes-for-failed-validation-or-invalid-duplicate).

Comment (beraldoleal) [reply main.py]: Agreed with @hdiogenes , please @macartur make use of 409 error.

Comment (macartur) [reply main.py]: Fixed.

**Pull request merged**